src/utils/fill_zero.jl

```diff
@@ -0,0 +1,187 @@
+"""
```

Comment:
**Organization** Functions related to filling dictionaries/dataframes with zeros